packages/map/README.md

```diff
@@ -41,13 +41,10 @@ const App = () => (
     {/* Credentials for maptiler */}
     credentials={{ key: 'Maptiler API secret key', map_id: 'Map style id' }}
 
-    {/* The initial center of the map */}
-    defaultView={[]}
+    {/* Max level of zoom. Can be used when there is only one marker on the map */}
```

Comment:
does it really only work if only one marker is on the map?

Reply:
it's probably for setting `default` zoom/viewbox when there is only 1 marker and nothing else to show